core/src/main/resources/schema/app/20201103154306_CB-9366_Create_load_balancer_metadata_table.sql

```diff
@@ -0,0 +1,62 @@
+-- // CB-9366 Create load balancer metadata table
+-- Migration SQL that makes the change goes here.
+
+CREATE SEQUENCE loadbalancer_id_seq START WITH 1
+  INCREMENT BY 1
+  NO MINVALUE
+  NO MAXVALUE
+  CACHE 1;
+
+CREATE TABLE loadbalancer (
+    id bigint default nextval('loadbalancer_id_seq'::regclass) not null
+      constraint loadbalancer_pkey
+        primary key,
+    dns character varying(255),
+    hostedzoneid character varying(255),
+    ip character varying(255),
```

Comment:
as per above, this may need to be a list of ips (but a short list), so may even be `ip_1`, `ip_2`, `ip_3`.

Reply:
See my previous comment about the ip addresses. If we need this to be a list when Azure is incorporated, I'll make a new SQL file to update the table.